src/kinetics/PlogRate.cpp

```diff
@@ -120,6 +116,12 @@ void PlogRate::setRates(const std::multimap<double, ArrheniusRate>& rates)
         j++;
         rates_.push_back(rate.second);
     }
+    if (!m_valid) {
+        // ensure that reaction rate can be evaluated (but returns NaN)
+        rates_.reserve(1);
+        pressures_[std::log(OneBar)] = {0, 1};
```

Comment:
Since there's only one rate at this pressure, I think the range should be `{0, 0}`. You can see that the case where there's only one rate at a pressure is handled specially in `PlogRate::evalFromStruct`, where these two values correspond to (`ilow1_` and `ilow2_`) or (`ihigh1_` and `ihigh2_`).

Reply:
thanks for explaining - fixed as recommended.